.opex/ecommerce-checkout/env/prod/config.yaml

```diff
@@ -10,4 +10,104 @@ action_groups:
 overrides:
   hosts: # Use these hosts instead of those inside the OpenApi spec
     - api.platform.pagopa.it
-  base_path: "ecommerce/checkout/v1"
\ No newline at end of file
+  base_path: "ecommerce/checkout/v1"
+  endpoints:
+    /payment-requests/{rpt_id}:
+      availability_threshold: 0.95 # Default: 99%
+      availability_evaluation_frequency: 30 # Default: 10
+      availability_evaluation_time_window: 50 # Default: 20
+      availability_event_occurrences: 3 # Default: 1
+      response_time_threshold: 2 # Default: 1
+      response_time_evaluation_frequency: 35 # Default: 10
+      response_time_evaluation_time_window: 55 # Default: 20
+      response_time_event_occurrences: 5 # Default: 1
+    /transactions:
+      availability_threshold: 0.95 # Default: 99%
+      availability_evaluation_frequency: 30 # Default: 10
+      availability_evaluation_time_window: 50 # Default: 20
+      availability_event_occurrences: 3 # Default: 1
+      response_time_threshold: 2 # Default: 1
+      response_time_evaluation_frequency: 35 # Default: 10
+      response_time_evaluation_time_window: 55 # Default: 20
+      response_time_event_occurrences: 5 # Default: 1
+    /transactions/{transactionId}:
```

Comment:
All urls are correctly listed here. 
Just a point, we have a POST and DELETE with this path `/transactions/{transactionId}`
According to you this values will override both apis?

Reply:
Yes. 
Alerts generated by opex take into account all requests made to openapi paths with no discrimination against http method.